### ISSUE TITLE
Fix issue with `clamdscan` not scanning correct files

### ISSUE THREAD

**Issue by jerekshoe** [department-of-veterans-affairs/va.gov-team#47480]
## Background
We are using an antivirus called `clamav` in order to ensure that any file uploaded to `vets-api` do not contain viruses. `clamav` is a command-line utility, and in order to call it in Rails, we are using a gem called `clam_scan` ([Github repo](https://github.com/jschroeder9000/clam_scan)). `clam_scan` provides us with the ability to supply keyword arguments that translate to the flags provided by the CLI. If you would like to provide arguments not provided by the gem, you can specify an array of custom arguments using the `custom_args` named argument **instead**.

The docs don't make it clear, but you can either opt to use the keyword arguments **OR** `custom_args`, but you **CANNOT** mix-and-match. We are attempting to provide a `location` named argument to specify the path of the file to scan, as well as `custom_args`, which is not supported by `clam_scan`. Because named arguments are ignored, `clamav` is scanning the entire root of `vets-api` instead of just the file we are targeting.

**TL;DR** Currently, we are scanning the entire `vets-api` directory for viruses instead of the uploaded files

## Problem code in `clam_scan`
https://github.com/jschroeder9000/clam_scan/blob/master/lib/clam_scan/request.rb#L102-L104

## Problem code in `vets-api`
https://github.com/department-of-veterans-affairs/vets-api/blob/d350d82c/lib/common/virus_scan.rb#L10-L11
